todolist.js

```diff
@@ -20,3 +20,5 @@ document.addEventListener("keydown", function (e) {
     addTasks();
   }
 });
+
+//js file ends here
```

Comment:
do we need this comment?

Reply:
No, we do not need this comment and this has been resolved.